pinot-query-runtime/src/test/resources/log4j2.xml

```diff
@@ -32,6 +32,12 @@
 <!--    <Logger name="org.apache.pinot.query" level="trace" additivity="false">-->
 <!--      <AppenderRef ref="console"/>-->
 <!--    </Logger>-->
+
+    <!-- Change level to DEBUG in order to log the optimization process -->
+    <logger name="org.apache.calcite.plan.RelOptPlanner" level="ERROR" additivity="false">
```

Comment:
Why not `INFO`?

Reply:
Probably info is fine, but I decided to use error because the root is defined as error as well and I don't want to increase the number of logs by accident in this PR.